login1/dbus.go

```diff
@@ -258,3 +258,8 @@ func (c *Conn) Subscribe(members ...string) chan *dbus.Signal {
 func (c *Conn) PowerOff(askForAuth bool) {
 	c.object.Call(dbusInterface+".PowerOff", 0, askForAuth)
 }
+
+// Suspend asks logind for a suspend optionally asking for auth.
+func (c *Conn) Suspend(askForAuth bool) {
```

Comment:
It would be better to return the inner `error` here, see https://github.com/coreos/go-systemd/issues/326.